NetworkConfigurations/TrafficControlAndPolicingSubCategories/DynamicSecureControlTraffic.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Dynamic Secure Control Traffic
+grand_parent: Network Configuration
+parent: Traffic Control and Policing
+nav_order: 4
+layout: default
+---
+
+# Dynamic Secure Control Traffic
```

Comment:
How did you test the sample config you've documented below is correct?

Reply:
There are no tests here, just the data for initial SCT values, similar to Marvell's documentation.

User configuration (read, write, disable) has been tested on hardware, and outputs have been noted accordingly.